wasm.js

```diff
@@ -45,6 +45,20 @@ async function createObjectURL(buf, mimeType) {
   return URL.createObjectURL(blob);
 }
 
+/**
+ * Sleep is used when awaiting for Go Wasm to initialize or waiting for remote response inside sdk.
+ * It uses the lowest possible sane delay time (via requestAnimationFrame).
+ * However, if the window is not focused, requestAnimationFrame never returns.
+ * A timeout will ensure to be called after 50 ms, regardless of whether or not
+ * the tab is in focus.
+ *
+ * @returns {Promise} an always-resolving promise when a tick has been
+ *     completed.
+ */
+ const sleep = (ms) => new Promise(res => {
```

Comment:
One suggestion, in case someone forgets to pass in a time:
```suggestion
 const sleep = (ms = 1000) => new Promise(res => {
```